Makefile

```diff
@@ -26,8 +26,8 @@ test-lmnet: build
 
 .PHONY: test-dlk
 test-dlk: build
-	# Run dlk test (only available on Jenkins)
-	docker run --rm -t -v /root/.ssh:/root/.ssh --net=host $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "apt-get update && apt-get install -y iputils-ping && cd dlk && python setup.py test"
+	# Run dlk test
+	docker run --rm -t -v $(HOME)/.ssh:/tmp/.ssh -e FPGA_HOST --net=host $(IMAGE_NAME):$(BUILD_VERSION) /bin/bash -c "cp -R /tmp/.ssh /root/.ssh && apt-get update && apt-get install -y iputils-ping && cd dlk && python setup.py test"
```

Comment:
Why do you volume from `$(HOME)/.ssh` to `/tmp/.ssh` instead of `/root/.ssh` ?

Reply:
To avoid SSH error: `Bad owner or permissions on /root/.ssh/config`
In the case of `-v ${HOME}/.ssh:/root/.ssh`, the owner of `/root/.ssh/config` is local user (e.g. `jenkins`), `ssh` raises an error and fails. So I put them on `/tmp/.ssh` and copy `/root/.ssh` as root owner permission.